### PR TITLE
 Fix the __enable_kvm wrong logic and add related test

### PR DESCRIPTION
**Pull request by Summer-Qing**

### CODE REVIEW

**Comment by emcinfrasim** [comment]
Can one of the admins verify this patch?

File: test/unit/test_model.py [hunk 1]
```diff
@@ -764,7 +764,92 @@ def test_monitor_fault_path_in_control_mode(self):
         except ArgsNotCorrect, e:
             assert "Path folder doesn't exist: /fake/path" in e.value
 
+	
+    def test_kvm_enabled_yaml_true_env_true(self):
+        if os.path.exists("/dev/kvm")== False:
```
Comment (turtle-fly) [inline test/unit/test_model.py]: use `if not os.path.exists("/dev/kvm")` instead

File: test/unit/test_model.py [hunk 2]
```diff
@@ -764,7 +764,92 @@ def test_monitor_fault_path_in_control_mode(self):
...
+        compute_info["kvm_enabled"] = "true"
+        workspace = "{}/{}".format(config.infrasim_home, node_info['name'])
+        compute = model.CCompute(compute_info)
+        compute.set_workspace(workspace)
+        compute.init()
+        compute.handle_parms()
+        assert "--enable-kvm" in compute.get_commandline()
+    
+    def test_kvm_enabled_yaml_true_env_False(self):
+        if os.path.exists("/dev/kvm")== True:
```
[the remark this answers is not in the document]
Comment (turtle-fly) [reply test/unit/test_model.py]: @Summer-Qing see this houndci comments~

File: test/unit/test_model.py [hunk 3]
```diff
@@ -764,7 +764,92 @@ def test_monitor_fault_path_in_control_mode(self):
         except ArgsNotCorrect, e:
             assert "Path folder doesn't exist: /fake/path" in e.value
 
+	
+    def test_kvm_enabled_yaml_true_env_true(self):
+        if os.path.exists("/dev/kvm")== False:
+            self.skipTest(' os.path.exists("/dev/kvm") == False, skip test_kvm_enabled_yaml_true_env_true')
```
Comment (turtle-fly) [inline test/unit/test_model.py]: Try to use human readable text in information, e.g. `OS disable KVM, skip.`

File: infrasim/model.py [hunk 4]
```diff
@@ -1782,10 +1782,15 @@ def precheck(self):
 
     @run_in_namespace
     def init(self):
-        if 'kvm_enabled' in self.__compute and not helper.check_kvm_existence():
+        if not helper.check_kvm_existence():
             self.__enable_kvm = False
+        elif 'kvm_enabled' in self.__compute and helper.check_kvm_existence():
```
Comment (turtle-fly) [inline infrasim/model.py]: This `elif` should imply `helper.check_kvm_existence() is True`, this code should be omitted.

**Comment by fub2** [comment]
Well done! @Summer-Qing

**Pull request merged**